PowerPointLabs/Test/UnitTest/SyncLab/SyncLabPositionTest.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Microsoft.Office.Interop.PowerPoint;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Generic;
+using PowerPointLabs.SyncLab.ObjectFormats;
+
+namespace Test.UnitTest.SyncLab
+{
+    [TestClass]
+    public class SyncLabPositionTest : BaseSyncLabTest
```

Comment:
If possible better have some. For UT, it's headless, supports mock class, and doesn't require each components to be wired tgt. For FT, it's really like an end-to-end integration test to show it's working.